dtbase/backend/README.md

```diff
@@ -24,22 +39,26 @@ then you would call the other end points with the following in the header of the
 Authorization: Bearer abc
 ```
 
-If your token expires, you can use the refresh token to get a new for some time still,
-by calling the `/auth/refresh` end point. This one requires setting you header like above, but
-using the refresh token (`xyz`) rather than the access token (`abc`).
+If your token expires, you can use the refresh token to get a new for some time still, by calling the `/auth/refresh` end point. This one requires setting you header like above, but using the refresh token (`xyz`) rather than the access token (`abc`).
 
-## Locations
+### Locations
 
 Locations can be defined using any combination of floating point, integer, or string variables.   These variables, known as _LocationIdentifiers_ must be inserted into the database before an actual _Location_ can be entered.  The set of _LocationIdentifiers_ that is sufficient to define a _Location_ is called a _LocationSchema_.   A _Location_ will therefore have a _LocationSchema_, and one _LocationXYZValue_ for each _LocationIdentifier_ within that schema (where _XYZ_ can be "Float", "Integer" or "String").
 
-## Sensors
+### Sensors
 
 The Sensor data model is as follows.   Every _Sensor_ has a _SensorType_ which in turn specifies the variable(s) it can measure - these are known as _SensorMeasures_.  Each _SensorMeasure_ specifies its datatype (float, int, string, or bool), and these are used to define the type of the corresponding _SensorXYZReadings_.   A _Sensor_ may also have a _SensorLocation_, which specifies a _Location_ as defined above, and a time window (possibly open-ended) when the sensor was at that location.
 
-## Models
+### Models
 
 TODO: Write a summary of how storing model data works.
```

Comment:
TODO here that we should complete before merging for completeness

Reply:
I added a couple of paragraphs here. There are no more TODO notes in the docs.